packages/typescript-estree/src/visitor-keys.ts

```diff
@@ -43,7 +43,7 @@ export const visitorKeys = eslintVisitorKeys.unionWith({
   BigIntLiteral: [],
   ClassProperty: ['decorators', 'key', 'typeAnnotation', 'value'],
   Decorator: ['expression'],
-  OptionalCallExpression: eslintVisitorKeys.KEYS.CallExpression,
```

Comment:
`eslintVisitorKeys.KEYS.CallExpression` = ['callee', 'arguments']
and should
`['callee', 'typeParameters', 'arguments']`